target/linux/ramips/rt288x/config-4.9

```diff
@@ -134,6 +134,7 @@ CONFIG_MTD_SPI_NOR=y
 CONFIG_MTD_SPLIT_FIRMWARE=y
 CONFIG_MTD_SPLIT_LZMA_FW=y
 CONFIG_MTD_SPLIT_UIMAGE_FW=y
+CONFIG_MTD_SPLIT_WRGG_FW=y
```

Comment:
Not tested yet, but I'm quite sure you need to add a `# CONFIG_MTD_SPLIT_WRGG_FW is not set` to `target/linux/generic/config-*`.

Otherwise building any other target or a different subtarget than rt288x should cause a build error because of the new config symbol.

Reply:
As the splitter was just extended, `CONFIG_MTD_SPLIT_WRGG_FW` was already part of `target/linux/generic/config-*`